.github/workflows/dotnetcoretest.yml

```
@@ -39,9 +39,9 @@ jobs:
         run: dotnet build --configuration Release --no-restore
       - name: Test (Linux)
         working-directory: ${{ runner.workspace }}/Lightning.NET
-        run: LD_LIBRARY_PATH=${{ runner.workspace}}/Lightning.NET/lmdb/libraries/liblmdb/:$LD_LIBRARY_PATH dotnet test -f net6.0 --no-restore --verbosity normal
+        run: LD_LIBRARY_PATH=${{ runner.workspace}}/Lightning.NET/lmdb/libraries/liblmdb/:$LD_LIBRARY_PATH dotnet test -f net7.0 --no-restore --verbosity normal
         if: matrix.os == 'ubuntu-latest'
       - name: Test (Windows or Mac)
         working-directory: ${{ runner.workspace }}/Lightning.NET
-        run: dotnet test -f net6.0 --no-restore --verbosity normal
+        run: dotnet test -f net7.0 --no-restore --verbosity normal
```

```suggestion
        run: dotnet test -f net7.0 --no-restore --no-build --verbosity normal
```

since you already restored, AND it's also already built in the previous step for all TFMs